server/migrations/functions.py

```diff
@@ -0,0 +1,29 @@
+# PostgreSQL functions that are applied, that needs to be both migrated and recreated
```

Comment:
I wasn't entirely sure how to play this. I wanted to do the ID generation in postgres to avoid having to retry saving customers if we got unlucky in the random ID generation. This is not really following any existing conventions. I put them here so that we can load them into the test runner as well, and it becomes fairly clear for the future how that works.